src/worker/models.ts

```diff
@@ -0,0 +1,78 @@
+import { WordList } from "@common/types";
```

Comment:
I appreciate this refactor! `storage.ts` was getting a bit too large 😓 

Could you please add a JSDoc comment to the file explaining what's supposed to be stored in here?

Reply:
not a problem, do you prefer another model file that is specifically for all of the storage models or adding JSDoc this this file is fine?

Reply:
I like option of having a file for _just_ the models we use in storage! This refactor helps keep the storage models in a cohesive place. I would just like a one line JSDoc at the top of the file like:

```typescript
/**
 * @file Interfaces used in storage
 */
```